src/_guides/libraries/library-tour.md

```diff
@@ -1035,6 +1035,38 @@ For more information, see
 [Exceptions](/guides/language/language-tour#exceptions)
 (in the language tour) and the [Exception API reference.][Exception]
 
+### Weak references and finalizers
+Dart is a [garbage-collected][] language,
+which means that any Dart object
+that isn't referenced by at least one other object
+will eventually be disposed by the garbage collector. 
+This default behavior may not be desireable in
+some scenarios involving native resources or 
+if the target object cannot be sub-classed.
```

Comment:
According to https://developers.google.com/style/word-list#subclass:

```suggestion
if the target object cannot be subclassed.
```

Reply:
Also cannot => can't